rust/src/compose.rs

```diff
@@ -135,10 +135,18 @@ struct ComposeImageOpts {
     /// Operate only on cached data, do not access network repositories
     offline: bool,
 
+    #[clap(long, conflicts_with_all = ["lockfiles", "lockfile_strict"])]
```

Comment:
Hmm, why do we add a conflict here? They can be used independently.

Reply:
If you write the lock file, you don't read it, so both `lockfiles`, and `lockfile_strict` make no sense

Reply:
The output lock file is complete and describes the final package set that was composed.

Input lockfiles constrain the set of possible solutions. They may be complete or incomplete.

See e.g. https://github.com/coreos/fedora-coreos-config/blob/rawhide/manifest-lock.overrides.yaml. That's the only lockfile we feed to rpm-ostree when building rawhide currently.

Reply:
You can read a partial lock file and write a full lock file, got it now and make sense, will send a patch shortly

Reply:
https://github.com/coreos/rpm-ostree/pull/5476